### ISSUE TITLE
DOC: warnings when compiling the documentation

### ISSUE THREAD

**Issue by itziakos** [simphony/simphony-common#103]
```python
/home/travis/build/simphony/simphony-common/simphony/io/h5_cuds.py:docstring of simphony.io.h5_cuds.H5CUDS.iter_particles:4: ERROR: Unexpected indentation.
/home/travis/build/simphony/simphony-common/simphony/io/h5_cuds.py:docstring of simphony.io.h5_cuds.H5CUDS.iter_particles:6: WARNING: Block quote ends without a blank line; unexpected unindent.
```

**Comment by itziakos** [comment]
more warnings
```
/var/build/user_builds/simphony/checkouts/0.1.0/doc/source/api/cuds.rst:12: WARNING: failed to import abstractparticles.ABCParticleContainer
/var/build/user_builds/simphony/checkouts/0.1.0/doc/source/api/cuds.rst:39: WARNING: failed to import particles.ParticleContainer
docstring of simphony.cuds.lattice.Lattice:6: SEVERE: Unexpected section title.
```

**Comment by tuopuu** [comment]
Fixed in #133
